bh/documentation/src/dropdown/Dropdown.vue

```diff
@@ -15,8 +21,10 @@
 <script>
 import Example from '@/Example';
 import Api from '@/Api';
-import SimpleDropDownExample from './examples/SimpleDropDownExample';
-import simpleDropDownExampleCode from '!!raw-loader!./examples/SimpleDropDownExample';
+import DropDownExample from './examples/DropDownExample';
```

Comment:
Rename files without Example in the end. All examples files should be in a examples dir, is not required this complement.